libs/sdk-ui-kit/src/@ui/UiTooltip/UiTooltip.tsx

```diff
@@ -0,0 +1,166 @@
+// (C) 2025 GoodData Corporation
+
+import {
+    useFloating,
+    autoUpdate,
+    flip,
+    offset,
+    arrow,
+    useHover,
+    useFocus,
+    useDismiss,
+    useInteractions,
+    FloatingPortal,
+    FloatingArrow,
+} from "@floating-ui/react";
+import React, { useRef, useMemo, useState } from "react";
+import { bem } from "@gooddata/sdk-ui-kit";
+import { UiTooltipProps } from "./types.js";
+import { UiTooltipAnchor } from "./UiTooltipAnchor.js";
+import { UiTooltipContent } from "./UiTooltipContent.js";
+import {
+    computeArrowOffset,
+    computeTooltipShift,
+    getDimensionsFromRect,
+    getDimensionsFromRef,
+    getFlipFallbackOrder,
+    getOppositeBasicPlacement,
+} from "./utils.js";
+import { ARROW_HEIGHT, ARROW_WIDTH, ARROW_EDGE_OFFSET } from "./constants.js";
+
+const { b, e } = bem("gd-ui-kit-tooltip");
+
+/**
+ * @internal
+ */
+export const UiTooltip: React.FC<UiTooltipProps> = ({
+    children,
+    arrowPlacement = "top",
+    hoverTrigger = false,
+    focusTrigger = false,
```

Comment:
could it be trriggered also by click for example? (is there some use case for it?) then it would make sense something like `triggerBy={["focus", "hover", "click"]}`